### PR TITLE
Geom

### PR DESCRIPTION
**Pull request by lbinria**
- Remove warnings !!!
 - Speed-up `vertex` function in volume and it doesn't need connectivity anymore
 - Add some features in geometry (polyhedron...)

## Note: 

I commented attributes in geogram IO. 
When connectivity attributes are not set, Graphite just doesn't display borders. I take a look into Graphite, there is a special condition to NOT connect facets on geogram format (we can request to remove that).

Take attention on commits that references #56.

### CODE REVIEW

File: tests/test-geom.cpp [hunk 1]
```diff
@@ -50,7 +54,7 @@ bool are_vec_equal(vec<n> a, vec<n> b, double eps) {
 	for (int i = 0; i < n; i++) {
 
 		// Check NaN / infinity consistency
-		if (std::isnan(a[i]) && std::isnan(b[i]) || std::isinf(a[i]) && std::isinf(b[i]))
+		if ((std::isnan(a[i]) && std::isnan(b[i])) || (std::isinf(a[i]) && std::isinf(b[i])))
```
Comment (ssloy) [inline tests/test-geom.cpp:57]: std::isfinite?

File: sandpit/informal_tests.cpp
Comment (ssloy) [inline sandpit/informal_tests.cpp:1]: pk ce fichier dans le commit?

Comment (lbinria) [reply sandpit/informal_tests.cpp:1]: a retirer en effet

File: tests/test-geom.cpp (hunk 1 above)
Comment (lbinria) [reply tests/test-geom.cpp:57]: Il faut vraiment savoir si a et b sont cohérent, si `std::nan(a) && std::inf(b)` => incohérence

**Pull request merged**